src/duplicacy_b2client.go

```diff
@@ -284,13 +293,28 @@ func (client *B2Client) ListFileNames(startFileName string, singleFile bool, inc
 	input["bucketId"] = client.BucketID
 	input["startFileName"] = startFileName
 	input["maxFileCount"] = maxFileCount
+	inputMapBool := true
```

Comment:
I think this will disappear if you take my suggestion to pass the HTTP verb in as an argument, but this isn't a very approachable variable name for anyone seeing this code without a pre-existing understanding of what it does (not least because a boolean's name should represent what will happen if it is true, but this variable's name implies that you'll pass a boolean in place of the input map argument, but then does the opposite). I hope you'll be eliminating it entirely (setting method = "HEAD" and input = nil, for example), but if you were going to keep it, a better name would be something like passBoolToInputParameter.

Reply:
all valid